### ISSUE TITLE
General task view improvements

### ISSUE THREAD

**Issue by zeileis** [cran-task-views/TeachingStatistics#1]
**Format:** We have converted the old XML-based task views file now to the new Markdown-based format. Please check the resulting .md file whether everything is in order or some corrections/improvements are needed. See the [corresponding documentation](https://github.com/cran-task-views/ctv/blob/main/Documentation.md) for more details.

**Update:** This would also be a good time to check the entire task whether it is up to date and correct. A write-up of the general goals and content guidelines is now summarized in the workflow for [proposing new task views](https://github.com/cran-task-views/ctv/blob/main/Proposal.md) (which was not yet available when you authored your task view). Improving your task view to conform more closely with these goals and guidelines, if necessary, would be highly appreciated.

**Co-maintainers:** In particular, we strongly encourage that all task views not only have a principal maintainer but also 1-5 co-maintainers in order to get more inspirations for extensions/improvements and share the maintenance workload. So please try to extend the list of co-maintainers, ideally also including female contributors and generally involving persons with diverse backgrounds.

**Comment by zeileis** [comment]
Paul, thanks for the follow-up. The minor improvements are much appreciated and I've turned your project "Public" now.

Regarding the co-maintainers: Sounds good, I'm very happy to provide feedback when you have some suggestions. Maybe someone from the MOSAIC or OpenIntro teams?

**Comment by zeileis** [comment]
@paulnorthrop, is there any follow-up on this? Might be a good time to invite a few possible co-maintainers before the next semesters hits...

**Comment by paulnorthrop** [comment]
@zeileis Yes, absolutely.  I invited someone a couple of weeks ago but I have not heard back yet.  I expect that they will get back to me soon, but I will invite others asap.

**Comment by zeileis** [comment]
Sounds good, thanks for the feedback!

**Comment by zeileis** [comment]
Paul @paulnorthrop it would be good to have some follow-up on this...

**Comment by paulnorthrop** [comment]
Achim @zeleis Apologies (again!) for being slow.  Someone has kindly agreed to join me as a co-maintainer and I have (a few minutes ago) invited her on GitHub page.  I think that I may have someone else lined up and will add them asap.

Dr Paul Northrop
Associate Professor
Department of Statistical Science
University College London
Gower Street
London
WC1E 6BT
Tel: +44 2080168461
Internal: 68461
E-mail: ***@***.***
http://www.homepages.ucl.ac.uk/~ucakpjn/

🕑 I work flexibly and sometimes send emails outside working hours. Please do not feel obliged to reply to this email outside of your normal working hours.
[cid:e1644528-b4f3-4597-8a07-db83cbc82dbe]
________________________________

**Comment by zeileis** [comment]
Thanks, Paul @paulnorthrop, that is great. Welcome on board, Mine @mine-cetinkaya-rundel!
